validator/testdata/feature_tests/amp-audioburst-audio.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
```

Comment:
Please move this file to `extensions/amp-audioburst-audio/0.1/test/`

Please add a copyright assignment statement in an HTML comment (see a similar test in another `.html` file). 

Please add an HTML comment describing the test.

Reply:
Done